cc/acl.go

```diff
@@ -40,89 +40,18 @@ type AddrsWithPagination struct {
 // args[1] - Know Your Client (KYC) hash
 // args[2] - user identifier
 // args[3] - user can do industrial operation or not (boolean)
+// args[4] - key type: ed25519, ecdsa, gost (optional)
```

Comment:
I suggest adding an additional method because the optional parameter might break services like Explorer.

Reply:
Can you explain this? I don't understand how an optional parameter can break an external service